BlazorMarkDownAppJwt/BlazorMarkDownAppJwt/Server/Controllers/AuthController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using BlazorMarkDownAppJwt.Shared;
+using BlazorMarkDownAppJwt.Server.Entities;
+using BlazorMarkDownAppJwt.Server.Services.Users;
+
+namespace BlazorMarkDownAppJwt.Server.Controllers
+{
+    [ApiController]
+    public class AuthController : ControllerBase
+    {
+        private string? CreateJWT(User user)
+        {
+            var secretkey = new SymmetricSecurityKey(System.Text.Encoding.UTF8.GetBytes("THIS IS THE SECRET KEY")); // NOTE: SAME KEY AS USED IN Program.cs FILE
+            var credentials = new SigningCredentials(secretkey, SecurityAlgorithms.HmacSha256);

```

Comment:
Usually it's a good pratice to isolate this code it's own class.

Reply:
I created a dedicated JWTHelper static class to manage Token generation in Backend Side.
https://github.com/clemalex06/BlazorMarkDown/pull/8/commits/614a92a5082bffb7dba45a98b6d1eef6dfdb552e